trafilatura/main_extractor.py

```diff
@@ -474,7 +483,10 @@ def handle_image(element: Optional[_Element]) -> Optional[_Element]:
     # post-processing: URLs
     src_attr = processed_element.get("src", "")
     if not src_attr.startswith("http"):
-        processed_element.set("src", re.sub(r"^//", "http://", src_attr))
+        if options is not None and options.url is not None:
+            processed_element.set("src", urljoin(options.url, src_attr))
```

Comment:
It would be better to define a variable like `link` and then just write `processed_element.set("src", link)`. It would probably more the code more readable.

Reply:
ok, fixed